code/mob/living/life/critical.dm

```diff
@@ -75,7 +75,11 @@
 				boutput(owner, "<span class='alert'>Your head [pick("feels like shit","hurts like fuck","pounds horribly","twinges with an awful pain")].</span>")
 				owner.losebreath+=10
 				owner.changeStatus("weakened", 3 SECONDS)
-		if (owner.health <= -100)
+		if (owner.health <= -100 && owner.reagents.has_reagent("synaptizine") && owner.reagents.has_reagent("atropine")) // Axes deathchance if you have synaptizine & atropine
+			var/deathchance = 0
+			if (prob(deathchance))
+				owner.death()
+		else if (owner.health <= -100)
```

Comment:
```suggestion
		if (owner.health <= -100 && !(owner.reagents.has_reagent("synaptizine") && owner.reagents.has_reagent("atropine"))) // Axes deathchance if you have synaptizine & atropine
```

Reply:
Only reason I didn't do something like that is because I wasn't sure how the logical operators worked.

From what I can tell, that would make atropine needed to even have a chance to die. Though I'm still unsure how the order of logical operators operate logically.

Reply:
Never mind, parentheses! Looks like it'd work!